portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,32 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+ 
+  public ArrayList<String> makeList(){
+    ArrayList<String> animals = new ArrayList<String>();
+    animals.add("pug");
+    animals.add("grizzly bear");
+    animals.add("sloth");
+    return(animals);
```

Comment:
If you had a bigger array or didn't want to call the `add` method every time, you could add an entire array all at once. Here's a few ways you could do it:

https://www.tutorialspoint.com/Conversion-of-Array-To-ArrayList-in-Java

This is totally cool though 👍